docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -254,22 +257,24 @@ So it is important to check that the code in the release branch is working befor
 Release
 -------
 
-.. warning:: The following steps should be carried out on a :doc:`Flocker development machine <vagrant>`.
-             Log into the machine using SSH agent forwarding so that you can push changes to GitHub using the keys from your workstation.
+#. Create and log in to a new :doc:`Flocker development machine <vagrant>` using SSH agent forwarding so that you can push changes to GitHub using the keys from your workstation.
```

Comment:
Perhaps the *new* should be emphasised in an accompanying note.

I find that `vagrant up` doesn't give me any warning if I have an existing, halted, vagrant-dev machine, despite it not satisfying the minimum version requirement.

```
(virtualenvwrapper-FLOC-1343 ? =)[~/.../HybridLogic/flocker]$ vagrant box list
centos65-x86_64-20140116   (virtualbox, 0)
chef/centos-7.0            (virtualbox, 1.0.0)
clusterhq/fedora20-updated (virtualbox, 2014.09.19)
clusterhq/flocker-dev      (virtualbox, 0.3.2.324.g7987f9b)
clusterhq/flocker-tutorial (virtualbox, 0.3.2.1559.gca68f69)
clusterhq/flocker-tutorial (virtualbox, 0.3.2.doc1)
clusterhq/flocker-tutorial (virtualbox, 0.3.2.doc1.1666.gc6da06b)
clusterhq/flocker-tutorial (virtualbox, 0.3.2.doc2)
ubuntu/trusty64            (virtualbox, 14.04)
(virtualenvwrapper-FLOC-1343 ? =)[~/.../HybridLogic/flocker]$ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Clearing any previously set forwarded ports...
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
==> default: Forwarding ports...
    default: 22 => 2222 (adapter 1)
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
    default: Warning: Connection timeout. Retrying...
==> default: Machine booted and ready!
==> default: Checking for guest additions in VM...
==> default: Mounting shared folders...
    default: /vagrant => /home/richard/projects/HybridLogic/flocker
==> default: Machine already provisioned. Run `vagrant provision` or use the `--provision`
==> default: to force provisioning. Provisioners marked to run always will still run.

```